examples/example-android/app/src/main/java/uniffi/wysiwyg_composer/wysiwyg_composer.kt

```diff
@@ -42,15 +42,15 @@ open class RustBuffer : Structure() {
 
     companion object {
         internal fun alloc(size: Int = 0) = rustCall() { status ->
-            _UniFFILib.INSTANCE.ffi_wysiwyg_composer_e48b_rustbuffer_alloc(size, status).also {
```

Comment:
Is this generated code? Maybe it should not be checked in to git?

Reply:
I was adding it as part of the project because I had to change the name of the lib inside this file to make it work before. We can move it to the build folder as we used to do and generate it on each build.